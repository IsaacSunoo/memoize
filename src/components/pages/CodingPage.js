import React from 'react';
import Header from '../layout/Header';

function CodingPage() {
  return (
    <div>
        <Header />
        <h1>Coding Page... (coming soon)</h1>
    </div>
  )
}

export default CodingPage;