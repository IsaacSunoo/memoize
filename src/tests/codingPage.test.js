import React from 'react';
import CodingPage from '../components/pages/CodingPage';
import { shallow } from 'enzyme';

describe('CodingPage', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <CodingPage />
        )
    })

    it('should match the snapshot with all data passed in correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});