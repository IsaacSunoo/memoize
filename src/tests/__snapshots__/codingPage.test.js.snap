// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CodingPage should match the snapshot with all data passed in correctly 1`] = `
<div>
  <Header />
  <h1>
    Coding Page... (coming soon)
  </h1>
</div>
`;
